hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminClient.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.apache.hadoop.hbase.BalanceRequest;
```

Comment:
There's no `BalanceRSGroupRequest` ? I'm surprised to see a wire object without a POJO for use on the client. Reusing the POJO of one RPC for the other means the two APIs cannot evolve independently. We should not do this.

Reply:
I'll add this, but tbh this whole rsgroup split seems weird to me. I hear you on the ability for the two APIs to evolve independently, but in my very short experience so far, I'm only adding this here because I was asked to support the feature for rsgroup balance in the jira. So it seems like there is some desire for them to evolve _in lock step_, rather than independently.

Reply:
I'm not an expert in the RSGroupAdmin APIs, let me take a closer look...

I see what you mean. All these Java API methods accept a group name string in one form or another. In this context, your implementation using the base `BalanceRequest` makes sense. I guess it's better to keep it as you had it before my comment.

Reply:
Sounds good! I'll revert back to BalanceRequest so we can keep things simple for now. Thanks for the input